resolver/java/uni-resolver-web/src/main/java/uniresolver/web/servlet/ResolveServlet.java

```diff
@@ -83,7 +83,8 @@ protected void doGet(HttpServletRequest request, HttpServletResponse response) t
 
 		// write resolve result
 
-		if (request.getHeader("Accept").contains(DIDDocument.MIME_TYPE) && resolveResult.getDidDocument() != null) {
+		String acceptHeader = request.getHeader("Accept");
+		if ((acceptHeader == null || acceptHeader.contains(DIDDocument.MIME_TYPE)) && resolveResult.getDidDocument() != null) {
```

Comment:
Related: https://github.com/w3c/did-core/pull/396#issuecomment-695258183

`accept: application/json, text/plain, */*` seems like this logic won't work as expected... but I am not familiar enough with Java to know for sure... 

is `acceptHeader` a set with values [`application/json`, `text/plain`, `*/*`] ?

Shouldn't `*/*` always yield a did document regardless of the document mimetype?

Reply:
Might be helpful to review https://www.w3.org/Protocols/rfc2616/rfc2616-sec14.html

> If no Accept header field is present, then it is assumed that the client accepts all media types. If an Accept header field is present, and if the server cannot send a response which is acceptable according to the combined Accept field value, then the server SHOULD send a 406 (not acceptable) response.

You are currently throwing a 500, which indicates somewhere else... 

https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/500

Reply:
I think a null value for an accept header should return a 406 not a 500... 500 means the server made a mistake... this is the client's fault for sending a null header.

Reply:
I agree, if there is an `Accept:` header with no value, it seems that no content type is acceptable to the client, and therefore the server must return 406.